### PR TITLE
MINT-1693 

### PR DESCRIPTION
**Pull request by ex1st0r**
…ced; Update FileSelect; fix GasSlider; ProfileContent updates imports, styles, fix open modal

### CODE REVIEW

File: packages/core/redux/persistAccount/actions.js [hunk 1]
```diff
@@ -116,20 +115,17 @@ export const resetPasswordAccount = (wallet, mnemonic, password) => async (dispa
 }
 
 export const createAccount = ({ name, password, privateKey, mnemonic, numberOfAccounts = 0, types = {} }) => async (dispatch, getState) => {
-  let wallet, hex = privateKey ? `0x${privateKey}`: null || hdkey.fromMasterSeed(bip39.mnemonicToSeed(mnemonic)).derivePath(WALLET_HD_PATH).getWallet().getPrivateKeyString() || ''
+  let wallet, hex = (privateKey ? `0x${privateKey}`: null) || hdkey.fromMasterSeed(bip39.mnemonicToSeed(mnemonic)).derivePath(WALLET_HD_PATH).getWallet().getPrivateKeyString() || ''
```
Comment (ozalexo) [inline packages/core/redux/persistAccount/actions.js]: Suggestion (`let wallet` has been moved below):
```
let hex = privateKey ? `0x${privateKey}`: null
hex = hex || hdkey.fromMasterSeed(bip39.mnemonicToSeed(mnemonic)).derivePath(WALLET_HD_PATH).getWallet().getPrivateKeyString() || ''
```

File: packages/core/redux/persistAccount/actions.js [hunk 2]
```diff
@@ -116,20 +115,17 @@ export const resetPasswordAccount = (wallet, mnemonic, password) => async (dispa
...
 
   // const web3 = new Web3()
   // web3Provider.reinit(web3, web3Utils.createStatusEngine(settings))
   // web3Provider.resolve()
-  console.log('hex', hex, mnemonic, privateKey, password)
-
 
   const accounts = new Accounts()
 
   wallet = await accounts.wallet.create(numberOfAccounts)
```
Comment (ozalexo) [inline packages/core/redux/persistAccount/actions.js]: Moved from the line 118 to here (added 'let'):
``` 
let wallet = await accounts.wallet.create(numberOfAccounts)
```

**Review by ozalexo — APPROVED** [review]
LGTM
Suggestion may be safely ignored

File: packages/login/network/ProfileService.js [hunk 3]
```diff
@@ -6,7 +6,7 @@
 import EventEmitter from 'events'
 import axios from 'axios'
 
-const PROFILE_BACKEND_REST_URL = 'https://backend.profile.tp.ntr1x.com'
+const PROFILE_BACKEND_REST_URL = 'http://localhost:3000'
```
Comment (e-maximus) [inline packages/login/network/ProfileService.js]: Revert to https://backend.profile.tp.ntr1x.com

**Pull request merged**